testcontainers/src/clients/http.rs

```diff
@@ -33,46 +32,49 @@ struct Client {
     created_networks: RwLock<Vec<String>>,
 }
 
-impl fmt::Debug for Http {
-    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
-        f.debug_struct("Http").finish()
-    }
+#[async_trait]
+pub trait RunViaHttp<I: Image> {
+    async fn start(self) -> ContainerAsync<I>;
 }
 
-impl Default for Http {
-    fn default() -> Self {
-        Self::new()
+#[async_trait]
+impl<I> RunViaHttp<I> for I
+where
+    I: Image,
+    I::Args: Default,
+{
+    async fn start(self) -> ContainerAsync<I> {
```

Comment:
This is an helper implementation that allow you to start container directly from Image without needing to wrap image with RunnableImage::from